node-benchmark-compare.js

```diff
@@ -10,16 +10,39 @@ if (process.argv.length !== 3) {
   process.exit(1);
 }
 
+function parseCsvLine(data) {
+  let state = 0;
+  let escaped = false;
+  let tmp = '';
+  const items = [];
+
+  for (const char of data) {
+    if (char === '\\' && (escaped = !escaped)) continue;
+
+    if (state === 0 && /\s/.test(char)) {
+    } else if (char === '"' && state === 0) state = 1;
```

Comment:
```suggestion
    if (state === 0 && /\s/.test(char)) {
      // Ignore whitespaces
    } else if (char === '"' && state === 0) state = 1;
```

Or any other comment explaining the empty block